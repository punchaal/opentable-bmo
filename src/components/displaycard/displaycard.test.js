import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import DisplayCard from './displaycard.component';
import { Provider } from 'react-redux';
import store from '../../redux/store';

describe('Card component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <Provider store={store}>
        <DisplayCard />
      </Provider>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <DisplayCard name='Card' />{' '}
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
