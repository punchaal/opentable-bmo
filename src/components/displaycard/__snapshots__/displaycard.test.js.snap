// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component renders the UI as expected 1`] = `
Array [
  <ul
    aria-live="polite"
    className="grid-container"
    id="restaurant-results"
  />,
  " ",
]
`;
